cwms-data-api/src/test/java/cwms/cda/api/location/kind/OutletControllerTestIT.java

```diff
@@ -0,0 +1,336 @@
+            PROJECT_1_ID.getName() + "-CG20", OUTLET_KIND);
+    private static final Outlet EXISTING_CONDUIT_GATE_OUTLET = buildTestOutlet(EXISTING_CONDUIT_GATE, PROJECT_LOC,
+                                                                               CONDUIT_GATE_RATING_GROUP);
+
+    private static final Location NEW_CONDUIT_GATE_2 = buildProjectStructureLocation(PROJECT_1_ID.getName() + "-CG30",
+                                                                                     OUTLET_KIND);
+    private static final Outlet NEW_CONDUIT_GATE_2_OUTLET = buildTestOutlet(NEW_CONDUIT_GATE_2, PROJECT_LOC,
+                                                                            CONDUIT_GATE_RATING_GROUP);
+    private static final CwmsId RENAMED_CONDUIT_GATE = new CwmsId.Builder().withName(
+            NEW_CONDUIT_GATE_2.getName() + "_Renamed").withOfficeId(OFFICE_ID).build();
+
+    @BeforeAll
+    public static void setup() throws Exception {
+        setupProject();
+        CwmsDatabaseContainer<?> databaseLink = CwmsDataApiSetupCallback.getDatabaseLink();
+        databaseLink.connection(c -> {
+            DSLContext context = getDslContext(c, OFFICE_ID);
+            OutletDao outletDao = new OutletDao(context);
+            try {
+                deleteLocation(context, NEW_CONDUIT_GATE_1.getOfficeId(), NEW_CONDUIT_GATE_1.getName());
+                deleteLocation(context, RENAMED_CONDUIT_GATE.getOfficeId(), RENAMED_CONDUIT_GATE.getName());
+                storeLocation(context, NEW_CONDUIT_GATE_2);
+                storeLocation(context, EXISTING_CONDUIT_GATE);
+                outletDao.storeOutlet(EXISTING_CONDUIT_GATE_OUTLET, false);
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+        }, CwmsDataApiSetupCallback.getWebUser());
+    }
+
+    @AfterAll
+    public static void tearDown() throws Exception {
+        CwmsDatabaseContainer<?> databaseLink = CwmsDataApiSetupCallback.getDatabaseLink();
+        databaseLink.connection(c -> {
+            DSLContext context = getDslContext(c, OFFICE_ID);
+            OutletDao outletDao = new OutletDao(context);
+
+            outletDao.deleteOutlet(EXISTING_CONDUIT_GATE.getOfficeId(), EXISTING_CONDUIT_GATE.getName(),
+                                   DeleteRule.DELETE_ALL);
+            deleteLocation(context, NEW_CONDUIT_GATE_1.getOfficeId(), NEW_CONDUIT_GATE_1.getName());
+            deleteLocation(context, NEW_CONDUIT_GATE_2.getOfficeId(), NEW_CONDUIT_GATE_2.getName());
+            deleteLocation(context, RENAMED_CONDUIT_GATE.getOfficeId(), RENAMED_CONDUIT_GATE.getName());
+            deleteLocation(context, EXISTING_CONDUIT_GATE.getOfficeId(), EXISTING_CONDUIT_GATE.getName());
+        }, CwmsDataApiSetupCallback.getWebUser());
+        tearDownProject();
+    }
+
+//    @Disabled("Disabled due to a DB issue.  See https://jira.hecdev.net/browse/CWDB-296")
```

Comment:
remove commented out annotation

Reply:
That needs to stay in, the test won't work until we get that jira ticket resolved.